.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: ci
+
+on:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

Comment:
Would you be open to using v4 instead?

Reply:
I only ask because I saw the deprecation warning under the Checks tab.

Reply:
No


Just kidding, thank you for letting me know! :smile: